### ISSUE TITLE
Add an explanation about the TeamCity build error when submitting pull requests

### ISSUE THREAD

**Issue by esotericAltruist** [keymanapp/keyman#11848]
### Is your feature request related to a problem? Please describe.

It was a little confusing when I submitted my first pull request and immediately saw that I failed 1 of 2 checks when git processed my request. While I knew that the team would review my request, I wanted to know why I'd failed the check / if there was anything I needed to do to fix it. But trying to click on the "Details" link only brought me to a login page for TeamCity, so I couldn't view any of the information for the error.

### Describe the solution you'd like

Adding a small bit of text to the [Pull Request Tutorial](https://help.keyman.com/developer/keyboards/github/step-3) about the error and/or adding it to the comment left by keyman-server on the request itself would be helpful. Even something simple like "TeamCity will automatically fail to build, but this is a default behavior. The Keyman team will manually create the build when they review your request." ...would be helpful.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [X] Other - give details at bottom of form

### Target application

github.com

**Comment by ermshiperete** [comment]
Do you have a link to the PR?

Usually the text "This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR." is added if a PR is from an outside contributor and can't be built automatically.

**Comment by esotericAltruist** [comment]
> Do you have a link to the PR?
> 
> Usually the text "This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR." is added if a PR is from an outside contributor and can't be built automatically.

Yep, here's the link: https://github.com/keymanapp/keyboards/pull/2835
This is all that keyman-server comments, and then the following error appears:
![image](https://github.com/keymanapp/keyman/assets/29443515/82a4ca91-3b25-4736-9da2-9b5a95416f16)
![image](https://github.com/keymanapp/keyman/assets/29443515/d720469f-8fea-457a-a258-ae501e7a454f)
